pom.xml

```diff
@@ -45,7 +45,7 @@
     <maven.deploy.version>2.8.2</maven.deploy.version>
```

Comment:
can we have a sample entry in the above excluded-tests.txt file?
Perhaps create a new test simply for the purpose of always excluding it via this file?
Also, does the github action mechanism append excluded tests entries, or does it overwrite the file?